.github/workflows/release.yml

```diff
@@ -130,6 +135,45 @@ jobs:
               --docker-password $DOCKERHUB_TOKEN \
               --force \
               --branch $TARGET_BRANCH
+      - name: Run customer 1 workflows
+        if: ${{ github.event.inputs.should-run-customer-tests == 'true' }}
+        env:
+          CUSTOMER1_AUTH_TOKEN: ${{ secrets.CUSTOMER1_TOKEN }}
+          CUSTOMER1_PROJECT: ${{ secrets.CUSTOMER1_PROJECT }}
+          CUSTOMER1_WORKFLOW1_NAME: ${{ secrets.CUSTOMER1_WORKFLOW1_NAME }}
+          CUSTOMER1_WORKFLOW2_NAME: ${{ secrets.CUSTOMER1_WORKFLOW2_NAME }}
+        run:
+          export PATH="$PATH:$HOME/.local/bin"
+
+          # set up DNAnexus production environment for customer 1
+          dx login --noprojects --token CUSTOMER1_AUTH_TOKEN
+          dx select $CUSTOMER1_PROJECT
+
+          # download, compile, and run workflows
+          dx download -r /workflow_1
+          cd workflow_1
+          $workflow1=$(java -jar ./dxCompiler-${{ github.event.inputs.release-version }}.jar compile \
```

Comment:
How does the dxCompiler .jar exist -- from being compiled in an earlier GHA step?

Reply:
Why not put compilation outputs and run outputs into a folder with the timestamp, the way the current script does? That makes it easier to debug a specific run, or clean up the output from older runs.

Reply:
1) yes
2) done, thanks